appsignals-tests/contract-tests/src/test/java/software/amazon/opentelemetry/appsignals/test/jdbc/operationtests/DBOperation.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package software.amazon.opentelemetry.appsignals.test.jdbc.operationtests;
+
+public enum DBOperation {
+  CREATE_DATABASE("CREATE DATABASE", "testdb2"),
+  SELECT("SELECT", "testdb");
```

Comment:
I tend to agree with Tiago here.

We can see that something is wrong, but the way you are forced to create this abstraction and by the way you are forced to couple the operations `CREATE DATABASE` or `SELECT` to the database name `testdb2` or `testdb`.

However the problem is not here, but in the way we write the test, this is just the symptom of the problem, you should be able to remove this enum completely without loosing anything.